conda/cli/main_list.py

```diff
@@ -263,8 +273,14 @@ def print_explicit(prefix, add_md5=False, remove_auth=True):
             continue
         if remove_auth:
             url = common_url.remove_auth(common_url.split_anaconda_token(url)[0])
-        md5 = prefix_record.get("md5")
-        print(url + (f"#{md5}" if add_md5 and md5 else ""))
+        if add_md5:
+            md5 = prefix_record.get("md5")
+            print(url + (f"#{md5}" if md5 else ""))
+        elif add_sha256:
+            sha256 = prefix_record.get("sha256")
+            print(url + (f"#{sha256}" if sha256 else ""))
```

Comment:
```suggestion
        if add_md5 or add_sha256:
            hash_key = "md5" if add_md5 else "sha256"
            hash_value = prefix_record.get(hash_key)
            print(url + (f"#{hash_value}" if hash_value else ""))
```
Would this be code golf?